diff --git "a/C:\\Users\\CLINT~1.BAN\\AppData\\Local\\Temp\\TortoiseGit\\Ogr7F75.tmp\\OgreD3D9Texture-b872786-left.cpp" "b/F:\\meridian59-dotnet\\Engine\\src\\ogre-1.10.0\\RenderSystems\\Direct3D9\\src\\OgreD3D9Texture.cpp"
index d29afee..d40c214 100644
--- "a/C:\\Users\\CLINT~1.BAN\\AppData\\Local\\Temp\\TortoiseGit\\Ogr7F75.tmp\\OgreD3D9Texture-b872786-left.cpp"
+++ "b/F:\\meridian59-dotnet\\Engine\\src\\ogre-1.10.0\\RenderSystems\\Direct3D9\\src\\OgreD3D9Texture.cpp"
@@ -1662,8 +1662,8 @@ namespace Ogre
         // Hacky override - many (all?) cards seem to not be able to autogen on 
         // textures which are not a power of two
         // Can we even mipmap on 3D textures? Well
-        if ((mWidth & mWidth-1) || (mHeight & mHeight-1) || (mDepth & mDepth-1))
-            return false;
+        //if ((mWidth & mWidth-1) || (mHeight & mHeight-1) || (mDepth & mDepth-1))
+        //    return false;
 
         if (rkCurCaps.Caps2 & D3DCAPS2_CANAUTOGENMIPMAP)
         {
